Formula/zlib-ng.rb

```diff
@@ -0,0 +1,42 @@
+class ZlibNg < Formula
+  desc "Replacement of zlib with optimizations for 'next generation' systems"
+  homepage "https://github.com/zlib-ng/zlib-ng"
+  url "https://github.com/zlib-ng/zlib-ng/archive/2.0.1.tar.gz"
+  sha256 "8599893f9b78bf979c1a1d6549b730367c9186560c6879590354998cc55428cf"
+  license "Zlib"
+
+  depends_on "cmake" => :build
```

Comment:
You don't seem to be using this in the build.

Reply:
Ah, thanks! 
Forgot to delete it after experimenting with building with `cmake`.